### PR TITLE
Clean up code

### PR DESCRIPTION
**Pull request by indiesquidge**

### CODE REVIEW

File: queue/queue.c [hunk 1]
```diff
@@ -9,94 +9,81 @@
...
+
+  /* malloc array */
+  q->array = malloc(sizeof(queue_node) * (q->maxSize));
+  if(!(q->array)) {
+    perror("Error on queue Malloc");
+    return QUEUE_FAILURE;
+  }
+
+  /* Set to NULL */
+  for(i=0; i < q->maxSize; ++i) {
```
Comment (ianks) [inline queue/queue.c:35]: Let's space these suckers out.

`for (i = 0; i < q->maxSize; i++) {`

Comment (ianks) [inline queue/queue.c:35]: While we at it, let's get rid of this camelCase biz.

File: queue/queue.c [hunk 2]
```diff
@@ -9,94 +9,81 @@
...
-    }
-	
-    ret_payload = q->array[q->front].payload;
-    q->array[q->front].payload = NULL;
-    q->front = ((q->front + 1) % q->maxSize);
-
-    return ret_payload;
+void* queue_pop(queue* q) {
+  void* ret_payload;
+  if(queue_is_empty(q)) {
```
Comment (ianks) [inline queue/queue.c]: Unnecessary bracks. Space between variable declaration.

File: util/util.c [hunk 3]
```diff
@@ -7,77 +7,74 @@
...
+    return UTIL_FAILURE;
+  }
+
+  /* Loop Through result Linked List */
+  for(result=headresult; result != NULL; result = result->ai_next) {
+    /* Extract IP Address and Convert to String */
+    if(result->ai_addr->sa_family == AF_INET) {
+      /* IPv4 Address Handling */
+      ipv4sock = (struct sockaddr_in*)(result->ai_addr);
+      ipv4addr = &(ipv4sock->sin_addr);
```
Comment (ianks) [inline util/util.c:45]: Needs some logical spacing.

File: util/util.c [hunk 4]
```diff
@@ -7,77 +7,74 @@
...
+      if(!inet_ntop(result->ai_family, ipv4addr, ipv4str, sizeof(ipv4str))) {
+        perror("Error Converting IP to String");
+        return UTIL_FAILURE;
+      }
+      #ifdef UTIL_DEBUG
+      fprintf(stdout, "%s\n", ipv4str);
```
Comment (ianks) [inline util/util.c]: Indent inner `#ifdef` if we are going to use it.

File: util/util.c [hunk 5]
```diff
@@ -7,77 +7,74 @@
...
+      #endif
+      strncpy(ipstr, "UNHANDELED", sizeof(ipstr));
+      ipstr[sizeof(ipstr)-1] = '\0';
     }
+    else {
+      /* Unhandlded Protocol Handling */
+      #ifdef UTIL_DEBUG
+      fprintf(stdout, "Unknown Protocol: Not Handled\n");
+      #endif
+      strncpy(ipstr, "UNHANDELED", sizeof(ipstr));
```
Comment (ianks) [inline util/util.c]: typo.

File: util/util.c (hunk 3 above)
Comment (indiesquidge) [inline util/util.c:45]: What does that even mean?

Comment (ianks) [inline util/util.c:45]: Just some newlines to decrease the 'wall of text' factor.

File: queue/queue.c (hunk 1 above)
Comment (indiesquidge) [inline queue/queue.c:35]: I don't think we should be worrying too much about this. The spacing is like this for all of the files, and I don't really feel like it's a problem. I don't really want to go in file by file and space out every for, while, if, else, etc. that there could be. The camel case is ugly, but still. Not really a problem. This "code fix" was more of me getting the indentation and more severe spacing laid out correctly.

**Pull request merged**